runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -2285,14 +2285,15 @@ MM_CopyForwardScheme::updateMarkMapAndCardTableOnCopy(MM_EnvironmentVLHGC *env,
 MMINLINE void
 MM_CopyForwardScheme::scanOwnableSynchronizerObjectSlots(MM_EnvironmentVLHGC *env, MM_AllocationContextTarok *reservingContext, J9Object *objectPtr, ScanReason reason)
 {
-	if (SCAN_REASON_COPYSCANCACHE == reason) {
-		addOwnableSynchronizerObjectInList(env, objectPtr);
-	} else if (SCAN_REASON_PACKET == reason) {
-		if (isObjectInEvacuateMemoryNoCheck(objectPtr)) {
+	if (scanMixedObjectSlots(env, reservingContext, objectPtr, reason)) {
+		if (SCAN_REASON_COPYSCANCACHE == reason) {
```

Comment:
let's put a quick comment that we want to avoid double adding to the list in case of abort/rescan